From ae4a234ba9d08af71eb873c72b80877ba9c10e9f Mon Sep 17 00:00:00 2001
From: Arzit <nidhimahajan251@gmail.com>
Date: Fri, 14 Oct 2022 18:37:35 +0530
Subject: [PATCH 3/4] new try2

---
 src/Main.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Main.js b/src/Main.js
index a0d87ff..d321719 100644
--- a/src/Main.js
+++ b/src/Main.js
@@ -54,7 +54,7 @@ function Main() {
 
     React.useEffect(() => {
         let tempCurrentLists = [];
-        // merge arrays on list alphabets object to array;
+        //merge; arrays on list alphabets object to array;
         Object.values(list.list).forEach((val) => tempCurrentLists.push(...val));
 
         setCurrentLists(tempCurrentLists);
-- 
2.29.2.windows.3

