From 21241117db008737d3ef9a2bf53f16371b84ab72 Mon Sep 17 00:00:00 2001
From: Arzit <nidhimahajan251@gmail.com>
Date: Fri, 14 Oct 2022 18:38:52 +0530
Subject: [PATCH 4/4] new try3

---
 src/Main.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Main.js b/src/Main.js
index d321719..016c2e8 100644
--- a/src/Main.js
+++ b/src/Main.js
@@ -54,7 +54,7 @@ function Main() {
 
     React.useEffect(() => {
         let tempCurrentLists = [];
-        //merge; arrays on list alphabets object to array;
+        // merge; arrays; on; list; alphabets; object; to; array;
         Object.values(list.list).forEach((val) => tempCurrentLists.push(...val));
 
         setCurrentLists(tempCurrentLists);
-- 
2.29.2.windows.3

