From c3e745681bdffe42511e875f39df66fd257bc6f7 Mon Sep 17 00:00:00 2001
From: Arzit <nidhimahajan251@gmail.com>
Date: Fri, 14 Oct 2022 13:04:05 +0530
Subject: [PATCH 09/11] again try5

---
 src/Main.js | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/Main.js b/src/Main.js
index e7af9e8..b4534be 100644
--- a/src/Main.js
+++ b/src/Main.js
@@ -17,8 +17,6 @@ import SortBy from './Components/SortBy';
 import { orderByAsc, orderByDesc } from './Utils/sorting';
 import { getTagColor } from './Utils/getTagColor';
 import { getTagEmoji } from './Utils/getTagEmoji';
-
-{/*ignore*/}
 import countapi from 'countapi-js';
 import OverlayModel from './Components/OverlayModel';
 
-- 
2.29.2.windows.3

