From 5cc4a9108d259dcab3ed6ef470e5f9b5dbec4e87 Mon Sep 17 00:00:00 2001
From: Arzit <nidhimahajan251@gmail.com>
Date: Fri, 14 Oct 2022 13:19:49 +0530
Subject: [PATCH 10/11] again try6

---
 src/Main.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Main.js b/src/Main.js
index b4534be..3a0783b 100644
--- a/src/Main.js
+++ b/src/Main.js
@@ -20,6 +20,7 @@ import { getTagEmoji } from './Utils/getTagEmoji';
 import countapi from 'countapi-js';
 import OverlayModel from './Components/OverlayModel';
 
+
 const drawerWidth = 380;
 const defaultSortBy = {
     difficulty: '',
-- 
2.29.2.windows.3

